decentraland/unity-renderer
# fix: race condition on PX

## What does this PR change?
There is a reported race condition with wearables. This PR fixes it.

## How to test the changes?
1. Launch the explorer with this link: https://play.decentraland.org/?GLOBAL_PX=urn%3Adecentraland%3Aentity%3Abafkreihj3zxbsd4ekz5ifgpwhqzuglod4s46353yis333c3w74uasgjgpu%3FbaseUrl%3Dhttps%3A%2F%2Fsdk-content-server.decentraland.org%2Fipfs%2F&realm=baldr&position=58%2C52&island=baldr9kdc
2. Do it many times :sweat_smile: reproducibility of the race condition is ~2/5
3. Ensure this doesn't happen:

![image](https://user-images.githubusercontent.com/42750/224710843-3e28d091-2a03-4e37-9e79-9bb79e70d292.png)
